lib/models/class_exercise.dart

```diff
@@ -1,9 +1,21 @@
+import 'package:timer_bloc/models/models.dart';
+
 class Exercise {
   Exercise(
     this.name,
     this.time,
   );
 
   String name;
-  List<int> time;
+  List<SetTimer> time;
+}
+
+class SetTimer {
```

Comment:
I propose to change this name because it does not correspond to its essence.

I suggest changing the name to: Approach